pipenv/utils.py

```diff
@@ -712,25 +713,21 @@ def convert_deps_to_pip(deps, project=None, r=True, include_index=False):
         vcs = maybe_vcs[0] if maybe_vcs else None
         if not any(key in deps[dep] for key in ['path', 'vcs', 'file']):
             extra += extras
+        if not isinstance(deps[dep], six.string_types):
+            editable = str(deps[dep].get('editable', '')).lower() == 'true'
```

Comment:
Doesn’t TOML have native boolean? I thought you could

```python
editable = bool(deps[dep].get('editable', False))
```

Also it might be better to check for `Mapping` instead of “not string”.